keylime/migrations/versions/8c0f8ded1f90_convert_allowlists_to_ima_policies.py

```diff
@@ -81,6 +81,7 @@ def upgrade_cloud_verifier():
                     logger.info("Runtime policy field '%s' not found in existing allowlist; using default value", key)
                 else:
                     new_ima_policy[key] = alist_json[key]
+        ima.trim_runtime_policy(new_ima_policy, False)
```

Comment:
I do not think that migration code can be changed

Reply:
This is to prevent writing a policy with v2 and then failing to run with the policy after downgrade to code that only supports v1 where we may check for a version number and not support a v2 policy.

Reply:
I have to say I haven't tested this and it's not so easy to test anyway but running into trouble with a v2 policy in the DB while keylime only understands v1 (after a downgrade) would be bad. Only a true v2 policy, one that uses the new features of the policy, in the DB should cause issues. Keylime should refuse to use it since keylime won't behave correctly: does not reject files as expected.